src/app/payments/helpers.ts

```diff
@@ -43,9 +44,12 @@ export const constructPaymentFlowBuilder = async <
   const lndService = LndService()
   if (lndService instanceof Error) return lndService
   const paymentBuilder = LightningPaymentFlowBuilder({
-    localNodeIds: lndService.listAllPubkeys(),
-    usdFromBtcMidPriceFn,
-    btcFromUsdMidPriceFn,
+    config: {
+      localNodeIds: lndService.listAllPubkeys(),
+      usdFromBtcMidPriceFn,
+      btcFromUsdMidPriceFn,
+    },
+    flaggedPubkeys: getPubkeysToSkipProbe(),
```

Comment:
All looks good except I don't understand the extra `config` key... `flaggedPubkeys` is also config isn't it?

Reply:
> All looks good except I don't understand the extra `config` key... `flaggedPubkeys` is also config isn't it?

Yup I noticed this literally after I pushed the last commit. Changed now.